src/data2img.py

```diff
@@ -40,20 +14,32 @@ def display_img(data):
 def save_img(data, file_name):
     if not os.path.exists(os.path.dirname(file_name)):
         os.makedirs(os.path.dirname(file_name))
-
     i = toimage(data[1])
     i.save(file_name)
 
 
-if __name__ == "__main__":
-    # convert image to data
-    # returns data in format [(expression, numpy array)]
-    img_data = csv2array(CSV_FILE_NAME)
-
-    # save images to disk
+def export_image(img_data):
     for i, img in enumerate(img_data):
         # save image with filename "/{expression}/{row_num}.png"
         save_img(img, DATA_OUTPUT_DIR + "/{}/{}.png".format(img[0], i))
-
     # display an example for demo purpose
     display_img(img_data[2])
+
+
+def show_zoomed_image(image_as_list ,zoom_ratio):
+    resized_list = imresize(image_as_list, zoom_ratio)
+    resized_img = toimage(resized_list)
+    resized_img.show()
+    return resized_img
+
+
+def get_zoomed_image(image_as_list ,zoom_ratio):
+    resized_list = imresize(image_as_list, zoom_ratio)
```

Comment:
may produce sharper image if set imresize(interp="bicubic"). Maths is a lost on me but wikipedia says so 

'In image processing, bicubic interpolation is often chosen over bilinear interpolation or nearest neighbor in image resampling, when speed is not an issue.' 
https://en.wikipedia.org/wiki/Bicubic_interpolation